src/Task/PhpCsFixerV2.php

```diff
@@ -76,7 +76,11 @@ public function run(ContextInterface $context)
         $arguments = $this->processBuilder->createArgumentsForCommand('php-cs-fixer');
         $arguments->add('--format=json');
         $arguments->add('--dry-run');
-        $arguments->addOptionalArgument('--allow-risky=%s', $config['allow_risky'] ? 'yes' : 'no');
+
+        if (null !== $config['allow_risky']) {
```

Comment:
We try to avoid these if statements inside the tasks. Can you move this nullable argument to a new `$arguments->add*` method?

Reply:
> We try to avoid these if statements inside the tasks. Can you move this nullable argument to a new $arguments->add* method?

@veewee Done!===